fileManagment/ViewController.swift

```diff
@@ -0,0 +1,184 @@
+                        showAlert(message: "تم انشاء ملف جديد بداخل المجلد المختار ", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+
+                        
+                    }else{
+                    
+                        showAlert(message: "الرجاء اختيار مجلد ليتم انشاء ملف بداخلة", title: "خطأ", actionTitle: "حسنا")
+
+                        
+                    }
+            
+                    
+                case 1:
+                    print("مجلد")
+                    let desiredFolderDirectory = appFilesDirectory?.appendingPathComponent(name)
+
+                    try   fileManager.createDirectory(at: desiredFolderDirectory!, withIntermediateDirectories: true, attributes: [:])
+                    
+                    showAlert(message: "تم انشاء مجلد جديد", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+
+                    
+                default:
+                    let desiredFolderDirectory = appFilesDirectory?.appendingPathComponent(name)
+
+                    try   fileManager.createDirectory(at: desiredFolderDirectory!, withIntermediateDirectories: true, attributes: [:])
+
+                    let dirForFile = appFilesDirectory?.appendingPathComponent(name).appendingPathComponent("Note.txt")
+
+                    let text = contentTextField.text!.data(using: .utf8)
+
+                    fileManager.createFile(atPath: dirForFile!.path, contents: text, attributes: nil)
+                    
+                  showAlert(message: "تم انشاء مجلد و ملف جديد", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+                    
+                }
+                
+                
+
+            } catch  {
+
+            }
+            relodFiles()
+            fileTableView.reloadData()
+
+        }else{
+
+        }
+        
+    }
+    
+    @IBAction func segmentUpdate(_ sender: Any) {
+        
+        if actionTypeSegment.selectedSegmentIndex == 1 {
+     
+            contentTextField.isHidden = true
+
+
+        }else{
+            contentTextField.isHidden = false
+        }
+        
+    }
+    
+    
+    func showAlert(message : String , title : String, actionTitle: String)  {
+        
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: actionTitle, style: .default, handler: { action in}))
+        self.present(alert, animated: true, completion: nil)
+        
+    }
+    
+}
+
+
+
+
+extension ViewController : UITableViewDelegate , UITableViewDataSource{
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return foldersArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        let fileCell = fileTableView.dequeueReusableCell(withIdentifier: "FileTableViewCell") as! FileTableViewCell
+        
+        fileCell.layer.cornerRadius = 15
```

Comment:
good styling Rayan! 
Note: you can remove this line from here and add it inside your cell class, in the `awakeFromNib` function